guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -28,32 +41,61 @@ ifdef::foreman-el,katello[]
 
 include::proc_configuring-repositories-el.adoc[]
 
-ifdef::katello[]
+include::snip_step-verify-enabled-repolist.adoc[]
+
 +
-. Enable `powertools` repository:
+. Enable the DNF modules:
```

Comment:
Should this be singular, since you only specify a single module?
```suggestion
. Enable the DNF module:
```

Or plural because the one you specify enables other modules it depends on?